src/applications/static-pages/sass/modules/_m-facility-events.scss

```diff
@@ -0,0 +1,20 @@
+.va-c-event-info {
```

Comment:
Could we put a class on the `dl` elements instead, and then use `> dd` and `> dt` for those styles? In my experience writing styles like this puts you in place over time where we keep having to override styles with more and more selectors and it becomes hard to untangle or modify.

Reply:
Moved this stuff to a class on the dl element.